javascript-mjs/draft.md

```diff
@@ -94,7 +94,9 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 The media type registrations herein are divided into two major categories: the sole media type "text/javascript" which is now in common usage, and all of the media types that are obsolete.
 
-For both categories, The ECMAScript media types are to be updated to point to a non-vendor specific standard undated specification of ECMAScript. In addition, a new file extension of .mjs is to be added to the list of file extensions with the restriction that it must correspond to the Module grammar of {{ECMA-262}}. Finally, the {{HTML}} specification is using "text/javascript" as the default media type of ECMAScript when preparing script tags; therefore, "text/javascript" has been moved intended usage from OBSOLETE to COMMON.
+For both categories, the ECMAScript media types are to be updated to point to a non-vendor-specific standard undated specification of ECMAScript. In addition, a new file extension of .mjs is to be added to the list of file extensions with the restriction that it must correspond to the Module grammar of {{ECMA-262}}. Finally, the {{HTML}} specification is using "text/javascript" as the default media type of ECMAScript when preparing script tags; therefore, "text/javascript" has been moved intended usage from OBSOLETE to COMMON.
+
+These changes are to be reflected in [the IANA Media Type registry](https://www.iana.org/assignments/media-types/media-types.xhtml).
```

Comment:
I'm using a Markdown link here, but this should probably be formatted differently.